packages/sdk/src/evm/core/classes/contract-deployer.ts

```diff
@@ -972,17 +994,36 @@ export class ContractDeployer extends RPCConnectionHandler {
           create2FactoryAddress,
         );
 
-        const transactionsToSend = deploymentInfo.map((i) => {
-          return i.transaction;
+        const transactionsToSend = deploymentInfo.filter(
+          (i) => i.transaction.data && i.transaction.data.length > 0,
+        );
+
+        const transactionsforDirectDeploy = transactionsToSend
+          .filter((i) => {
+            return i.type !== "infra";
```

Comment:
so forwarder,weth,clonefactory will get deployed together, everything else is deployed individually?

Reply:
yes